src/util/psbt/serialize.rs

```diff
@@ -20,3 +26,60 @@ pub trait Deserialize: Sized {
 }
 
 impl_psbt_de_serialize!(Transaction);
+
+impl Serialize for Script {
+    fn serialize(&self) -> Vec<u8> {
+        self.to_bytes()
+    }
+}
+
+impl Deserialize for Script {
+    fn deserialize(bytes: &Vec<u8>) -> Result<Self, encode::Error> {
```

Comment:
Why do we take a `&Vec<u8>`? Has that always been the case? It should be `&[u8]`.